src/Symfony/Component/Validator/Tests/Constraints/CidrValidatorTest.php

```diff
@@ -118,7 +118,7 @@ public function testOutOfRangeNetmask(string $cidr, ?string $version = null, ?in
         $this
             ->buildViolation('The value of the netmask should be between {{ min }} and {{ max }}.')
             ->setParameter('{{ min }}', $cidrConstraint->netmaskMin)
-            ->setParameter('{{ max }}', $cidrConstraint->netmaskMax)
+            ->setParameter('{{ max }}', $maxExpected)
```

Comment:
if we want to keep the nullable arg, this should likely be:
```suggestion
            ->setParameter('{{ max }}', $maxExpected ?? $cidrConstraint->netmaskMax)
```